samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -69,21 +78,25 @@ Hosting your image on an AWS S3 Server
    It is located in the :file:`zephyr` subfolder of your build directory.
 #. Click the file you uploaded in the bucket and check the :guilabel:`Object URL` field to find the download URL for the file.
 
-When specifying the image file, use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL for the download hostname.
-Make sure to not include ``https``.
-Specify the file name as the remaining part of the URL.
+Remember to do the following when specifying the filenames:
+
+* Use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL as the hostname of the server sourcing the images, without including ``https://``.
```

Comment:
`sourcing` -> `hosting`?

Reply:
Updated this and the other one to `hosting`